setup.cfg

```diff
@@ -0,0 +1,34 @@
+##########################
+# Setup.py Configuration #
+##########################
+[metadata]
+name = metabolomics_spectrum_resolver
+version = 0.0.1-dev
+long_description = file: README.md
+long_description_content_type = text/markdown
+
+[options]
+install_requires =
+    flask
+    spectrum_utils
+    requests
+    qrcode
+    requests_cache
+    numba
+    matplotlib
+    numpy
+    scipy
+    gunicorn
+    click
+
+# Where is my code
+packages = find:
+package_dir =
+    = src
+
```

Comment:
might need to include package data for the templates to work on pip install in not editable mode.